src/webapp/cypress/integration/public/login.js

```diff
@@ -30,8 +30,7 @@ sizes.forEach(size => {
                         cy.get('button[type=submit]').click()
                     })
 
-                    cy.url().should('include', 'dashboard')
-
+                    cy.get('[data-test=dashboard-index]').should('be.visible')
```

Comment:
url will include /dashboard before the page has loaded which causes us headaches from time to time.  dashboard-index being visible means we have loaded the page.

Reply:
Ideal 👍 Can we change this to asserting existence please?